meta-resin-common/recipes-connectivity/openvpn/openvpn/openvpn-resin.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=OpenVPN Application On Resin
-Requires=prepare-openvpn.service
-After=syslog.target network.target prepare-openvpn.service
+Requires=prepare-openvpn.service tmp.mount
+After=syslog.target network.target prepare-openvpn.service tmp.mount
```

Comment:
do we need this here also since prepare-openvpn.service already Requires and is After tmp.mount?